noir-projects/aztec-nr/aztec/src/macros/mod.nr

```diff
@@ -71,35 +71,35 @@ comptime fn generate_contract_interface(m: Module) -> Quoted {
 
     quote {
         pub struct $module_name {
-            target_contract: dep::aztec::protocol_types::address::AztecAddress
+            addr: dep::aztec::protocol_types::address::AztecAddress
```

Comment:
why `target_contract -> addr` ?

Reply:
Ah forgot to mention this. Now that we have completion hints in the LSP I started shortening some of these common names, as they show up all over the place. In this case it was the parameter for `at` that I should've renamed really, not also the struct field.

![image](https://github.com/user-attachments/assets/2f8a38e6-e0cd-44c8-b36f-8104e037f563)